src/utils/file.utils.ts

```diff
@@ -0,0 +1,32 @@
+import fs from "node:fs";
+import path from "node:path";
+
+/**
+ * If paths is a sequence of segments, path.resolve is run
+ *
+ * If paths is a singular element, path.join is run
+ *
+ * if paths is empty, __dirname is used
+ *
+ * @param paths A sequence of paths or path segments.
+ * @throws {TypeError} if any of the arguments is not a string.
+ */
+export function readDirectory(
+  fileExtension: string = ".ts",
+  ...paths: string[]
+) {
```

Comment:
This interface is a bit weird. I wonder if instead we use a glob matching library instead:

```ts
readDirectory('./src/commands/*.ts')
```

Reply:
https://github.com/chainpatrol/discord-bot/pull/27/commits/eea90648b6d2bf148c1dc79edf7358feba98f4df